packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/contracts/ImageLibraryContract.kt

```diff
@@ -23,17 +24,48 @@ internal class ImageLibraryContract(
     Intent(Intent.ACTION_GET_CONTENT)
       .addCategory(Intent.CATEGORY_OPENABLE)
       .setType(input.options.mediaTypes.toMimeType())
+      .apply {
+        if (input.options.allowsMultipleSelection) {
+          this.putExtra(Intent.EXTRA_ALLOW_MULTIPLE, true)
+        }
+      }
 
   override fun parseResult(input: ImageLibraryContractOptions, resultCode: Int, intent: Intent?) =
     if (resultCode == Activity.RESULT_CANCELED) {
       ImagePickerContractResult.Cancelled()
+    } else if (input.options.allowsMultipleSelection) {
+      val uris = requireNotNull(intent).getClipDataUris()
+      val contentResolver = requireNotNull(appContextProvider.appContext.reactContext) { "React Application Context is null. "}.contentResolver
```

Comment:
Maybe extract it to a class property:
```kotlin
private val contentResolver: ContentResolver
  get() = requireNotNull(appContextProvider.appContext.reactContext) { 
      "React Application Context is null"
    }.contentResolver
```

Also, remove the dot and space from the end of the error message 😉